content/2017/03/17/serializing-things.rst

```diff
@@ -0,0 +1,229 @@
+public: yes
+author: kit
+tags: [Celery, Django, Serialization, Code]
+image:
+  - src: '2017/serializing-things/celery-man.jpg'
+summary: |
+  Transmitting objects between web processes and worker processes is a
+  requirement of many modern web apps. Given that the safest way to do so is to
+  use a serialization format that only includes primitive data types, how can
+  we send custom objects around?
+
+
+Serializing Things
+==================
+
+Many Django apps need some kind of out-of-band processing. Perhaps you have to
+make calls to a separate API server, and don't want to nest the latency of HTTP
+calls. Or perhaps you need to perform a slow and complex post-processing step
+on user-submitted data. Whatever it is, you can't just put everything in your
+web processes. Some things demand worker processes consuming tasks from a
+queue.
+
+Typically we use Celery_ for this, though increasingly `Django Channels`_ is
+emerging as a viable option. In either case, there are some similar issues that
```

Comment:
I find it hard to follow the "In either case..." sentence. Maybe the structure can be simplified?